nodejs/CONTRIBUTING.md

```diff
@@ -0,0 +1,32 @@
+# Contributing
+
+Join us at https://github.com/GoogleCloudPlatform/cloud-code-samples.
+
+## Run the Tests
+
+Some of the samples are tested by a dedicated [Cloud Build](https://cloud.google.com/cloud-build)  configuration.
+
+All sample tests are expected to take the following steps:
+
+* Build the service
+* Deploy to Cloud Run
```

Comment:
Since this is generic instruction applicable to all the samples for node, you should change it to reflect deployment to intended platform